examples/simple-filter-xml-mqtt/main.go

```diff
@@ -55,7 +55,10 @@ func main() {
 		Topic:     "sampleTopic",
 	}
 
-	pair := transforms.KeyCertPair{}
+	pair := transforms.KeyCertPair{
+        KeyFile:    "",
+        CertFile:   "",
+    }
```

Comment:
please run the go fmt before the code commit

Reply:
Fixed.